roles/activation_keys/README.md

```diff
@@ -0,0 +1,79 @@
+theforeman.foreman.activation_keys
+==================================
+
+This role creates and manages Activation Keys.
+
+Role Variables
+--------------
+
+This role supports the [Common Role Variables](https://github.com/theforeman/foreman-ansible-modules/blob/develop/README.md#common-role-variables).
+
+The main data structure for this role is the list of `activation_keys`. Each `activation_key` requires the following fields:
+
+- `name`: The name of the activation key.
+
+The following fields are required for an activation key but have defaults which make them optional for this role:
+
+- `lifecycle_environment`: Lifecycle Environment to assign to hosts registered with this activation key. Defaults to "Library".
+- `content_view`: Content View to assign to hosts registered with this activation key. Defaults to "Default Organization View".
+
+The following fields are optional in the sense that the API will use default values when they are omitted:
```

Comment:
I wonder if we should write "API" here, it's an implementation detail the user won't care about. How about "Foreman" or "server"?

Reply:
Good point. I think "server" is the clearest.

Reply:
There are a few more api strings in the following sections, mind tackling those too?